addons/project_todo/static/tests/delete_todo_stage_in_kanban_test.js

```diff
@@ -0,0 +1,79 @@
+/* @odoo-module */
+
+import { startServer } from "@bus/../tests/helpers/mock_python_environment";
+import { addModelNamesToFetch } from "@bus/../tests/helpers/model_definitions_helpers";
+
+import { start } from "@mail/../tests/helpers/test_utils";
+
+import {
+    editInput,
+    click,
+    getFixture,
+} from "@web/../tests/helpers/utils";
+import { setupViewRegistries } from "@web/../tests/views/helpers";
+
+addModelNamesToFetch(["project.project", "project.task", "project.task.type"]);
+
+let target;
+
+QUnit.module('kanban todo stage deletion', {
```

Comment:
The framework JS seems to define the base module as Views (when we test a view)

```suggestion
QUnit.module('Views', {
```